### PR TITLE
Auto update copyright year

### PR DESCRIPTION
**Pull request by MetRonnie**
Sorry, this is getting repetitive!

The middle commit contains the 400-odd removals of years from copyright notices in the code

### CODE REVIEW

File: t/rose-config-diff/00-basic.t [hunk 1]
```diff
@@ -305,7 +305,7 @@ file_cmp "$TEST_KEY.out" "$TEST_KEY.out" <<'__DIFF__'
     13	# description=Environment variable configuration
     14	[env]
     15	GEARSTICK_DECORATION=golfball
-    16	
+    16
```
Comment (oliver-sanders) [inline t/rose-config-diff/00-basic.t:308]: Will need to check these file comparisons still pass with whitespace stripped.

Comment (oliver-sanders) [reply t/rose-config-diff/00-basic.t:308]: Nope, the whitespace is functional.

Comment (MetRonnie) [reply t/rose-config-diff/00-basic.t:308]: D'oh

Comment (MetRonnie) [reply t/rose-config-diff/00-basic.t]: I've gone through every file where more than 1 line was changed and undone the whitespace changes in heredocs

Comment (oliver-sanders) [reply t/rose-config-diff/00-basic.t]: Thanks.

**Review by oliver-sanders — APPROVED** [review]

**Pull request merged**